Tabloid/client/src/components/TagCreate.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState  } from 'react';
+import { Form } from 'reactstrap';
+import { addTag } from '../modules/TagManagment';
+import Tags from './TagView';
+import { Link } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+export const CreateTag=()=>{
+    const history = useHistory();
+    const TagForm={
+        Name:''
+    }
+    const [tag, setTag] = useState(TagForm);
+
+    const handleInputChange = (evt) => {
+        const value = evt.target.value;
+        const key = evt.target.id;
+        const tagCopy = { ...tag };
+        tagCopy[key] = value;
+        setTag(tagCopy);
+
```

Comment:
Nitpicky, but this is a lot of whitespace that could be gotten rid of to make the formatting easier for you and others to read.

Reply:
@SchmidtFred  got ride of the white space